pyartcd/pyartcd/pipelines/scan_fips.py

```diff
@@ -13,42 +13,45 @@
 
 
 class ScanFips:
-    def __init__(self, runtime: Runtime, version: str, nvrs: Optional[list]):
+    def __init__(self, runtime: Runtime, nvrs: Optional[list]):
         self.runtime = runtime
-        self.version = version
         self.nvrs = nvrs
+        self.active_ocp_versions = [4.12, 4.13, 4.14, 4.15, 4.16, 4.17]
```

Comment:
This could be hard to discover when we have a new minor version. Thinking this should be controlled in a more visible config option.

Reply:
Yeah I'm not sure where I should keep this. Will move to artcommonlib for now. If we need to change it somewhere else, we can do so later